main.py

```diff
@@ -1,29 +1,38 @@
 import win32api
+import win32con
 import win32process
 import win32gui
 import argparse
 
 
+
 def build_parser():
     parser = argparse.ArgumentParser(description='This is a title changer')
-    parser.add_argument('pid', action="store", type=int)
+    parser.add_argument('proc_name', action="store", type=str)
     parser.add_argument('new_title', action="store", type=str)
 
     parse_results = parser.parse_args()
 
     return parse_results
 
 
-def find_it(hwnd, parser):
+def enum_handler(hwnd, parser):
     _, p = win32process.GetWindowThreadProcessId(hwnd)
 
-    if parser.pid == p:
+    try:
+        handle = win32api.OpenProcess(win32con.PROCESS_QUERY_INFORMATION | win32con.PROCESS_VM_READ, False, p)
+        proc_name = win32process.GetModuleFileNameEx(handle, 0)
+    except Exception as ex:
```

Comment:
You should specify which type of exception you're catchcing, a generic `except:` or `except Excetion:` expression, especially when you're not handling the exception, could have some unwanted side effects